src/System.Windows.Forms/src/System/Windows/Forms/DataGridView/DataGridView.Methods.cs

```diff
@@ -16353,7 +16363,7 @@ protected override void OnMouseDoubleClick(MouseEventArgs e)
                                 int columnIndex = (hti._typeInternal == DataGridViewHitTestTypeInternal.ColumnResizeRight) ? hti._col : hti._adjacentCol;
                                 if (columnIndex < Columns.Count)
                                 {
-                                    HandledMouseEventArgs hme = new HandledMouseEventArgs(e.Button, e.Clicks, e.X, e.Y, e.Delta, false /*defaultHandledValue*/);
+                                    HandledMouseEventArgs hme = new HandledMouseEventArgs(e.Button, e.Clicks, e.X, e.Y, e.Delta, defaultHandledValue: false);
```

Comment:
Let's convert these over to the preferred pattern since this code is being touched
```suggestion
                                    HandledMouseEventArgs hme = new(e.Button, e.Clicks, e.X, e.Y, e.Delta, defaultHandledValue: false);
```